# Duty Cycle (-1.0 ~ 1.0)
float64 value
